Gemfile

```diff
@@ -23,6 +22,9 @@ gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
 
+gem 'bootstrap-sass', '~> 3.2.0'
```

Comment:
Sass allows you to do cool things like nested styles and importing mixins. I doubt we'll make extensive use of its features but if you know CSS, you're good.